service/controller.go

```diff
@@ -332,6 +350,12 @@ func (s *service) CreateVolume(
 		// use the default if not set in the storage class
 		softGracePrd = SoftGracePrdParamDefault
 	}
+	// If value is passed in pvc than it should get precedence
```

Comment:
Pls create a new method by passing "params" and return the needed variables (after assigning values).

Reply:
Will be addressed in upcoming PR